docs/metrics/metrics-operators/rate.md

```diff
@@ -7,23 +7,27 @@ sidebar_label: rate
 
 The rate operator calculates the per-second rate of change between data points in the time series. There are two different modes in which the rate operator can work:
 
-* Regular `rate` (without `over` modifier) calculates the rate of change between consecutive data points. It divides the difference in values of consecutive data points by the difference in their timestamps (in milliseconds) and then multiplies the result by 1000 (to scale up the quantity from a per-millisecond rate to a per-second rate).
+* Regular `rate` (without `over` modifier) calculates the rate of change between consecutive data points. It divides the difference in values of consecutive data points by the difference in their timestamps in seconds.
 * `rate over TIME`, on the other hand, allows you to calculate an average rate of change over a specified time window. This is achieved by considering the first and the last metric value in the given time window, extrapolating to the ends of the window if necessary.
 
 The rate operator also assigns the value of the metric tag to be `rate($metric)` and the value of the unit metadata field to be `$unit/second` (for example, 1/second).
 
-
 ## Syntax
 
 ```sql
 rate [increasing | decreasing | counter] [[over] INTERVAL]
 ```
 where `INTERVAL` is the duration over which you want to calculate and average rate, in seconds (`s`) , minutes (`m`), hours (`h`), or days (`d`).
 
-You can use the `increasing` or `decreasing` option to make `rate` consider only positive or negative rate values, respectively. This functionality is useful when you are calculating the positive or the negative rate of change of a metric over time.
+When deciding which mode you should use, here are some guidelines:
+* You need to get an instant rate of a counter - use `rate counter`.
+* You need to get an average rate of a counter over an interval - use `rate counter over INTERVAL`.
+* You need to get an instant rate of a non-counter metric - use `rate`.
+* You need to get an average rate of a non-counter metric over an interval  - use `rate over INTERVAL`.
 
-When working with counters, you can also use a dedicated `counter` mode which accounts for counter resets and always returns non-negative rate values.
+The usage of `counter` mode is recommended for counter-like metrics, because it accounts for counter resets and always returns non-negative rate values
```

Comment:
```suggestion
The usage of `counter` mode is recommended for a counter, like metrics because it accounts for counter resets and always returns non-negative rate values.
```